Allow ops usage inside an unikernel

fabioDMFerreira opened the pull request:
Use current directory if there is no way of getting user home directory.

eyberg reviewed (approved):
that dirUnix function looks overly complicated for no reason but this seems fine to me

fabioDMFerreira commented:
I simplified and removed unused methods from homedir file.

The removed code is based on https://github.com/mitchellh/go-homedir. It has alternative methods to get the home directory in case of HOME environment variable not being present.

The current method uses the go native method https://golang.org/pkg/os/#UserHomeDir.
